src/constants/hotm.js

```diff
@@ -759,7 +759,7 @@ class PeakOfTheMountain extends Node {
     const output = [];
 
     for (let tier = 1; tier <= level; tier++) {
-      for (const [reward, qty] of Object.entries(rewards.potm[tier])) {
+      for (const [reward, qty] of Object.entries(rewards.hotm[tier])) {
```

Comment:
this `potm` was correct, if it creates other errors those needs to be fixed

Reply:
Yeah I'm aware as I said on discord they have updated the hotm tree. This fixes error so it's kinda a fix for it as well. Will update it tomorrow

Reply:
![image](https://user-images.githubusercontent.com/2744227/192893933-0cd74812-5a37-45c0-a405-4c0b0b05f595.png)

Oh shit missed this in patch notes... welp found why then

Reply:
Solved